### ISSUE TITLE
The value of loss function

### ISSUE THREAD

**Issue by MuZiGuYue** [flyingpot/pytorch_deephash#7]
Hello. When using AlexNet for classification, the accuracy is higher, but after adding the hidden layer, the accuracy is very low. Do you know the reason?The value of my loss function has been maintained at around 2. Did your loss function fall faster? Thank you.

**Comment by spoonnotfound** [comment]
I can't locate the problem with so few information. How and why did you add a hidden layer? Is the accuracy high when you do no edit in the code?
Yes, the loss should fall fast.

**Comment by MuZiGuYue** [comment]
My approach is that  using the model trained in the ImageNet dataset initializes the first 7 layers of the AlexNet model , then add a hidden layer between the 7th and 8th layers. This method keeps the value of the loss function decreasing, but the accuracy rate increases before 20 iterations and decreases after 20 iterations. Do you know the reason for this?

**Comment by spoonnotfound** [comment]
OK. Maybe the learning rate is too high for the new hidden layer. You can try to reduce the learning rate to solve it.

**Comment by MuZiGuYue** [comment]
Ok, there is another question: what is the accuracy of your final classification and mAP?

**Comment by spoonnotfound** [comment]
It is about 93% classification accuracy and about 94% mAP.

**Comment by MuZiGuYue** [comment]
OK, thank you very much
